test/generators/add-action/asset-compute.test.js

```diff
@@ -41,7 +41,7 @@ describe('prototype', () => {
 
 function assertGeneratedFiles (actionName) {
   const actionPath = `actions/${actionName}`
-  const testPath = 'test'
+  const testPath = `test/asset-compute/${actionName}`
```

Comment:
is it possible to have these tests actually run `aio app test`? (in the future after the aio cli plugin asset-compute works with the new aio)

I think that would have helped catch this bug earlier

Reply:
Not sure I follow exactly. Yes, if the tests are jest tests, these can be run automatically by `aio app test`. Right now it reads the `test` hook in the extension's `ext.config.yaml` and runs that (this is analogous to the old way where it ran your test via adding the command to `npm test`)

Reply:
Yes that was what I was picturing, running `aio app test` on the created project